examples/6field-simple/elm_6f.cxx

```diff
@@ -12,1677 +12,1682 @@
+      else
+        result[i] = limit;
+    }
+    mesh->communicate(result);
+    return result;
   }
-  mesh->communicate(result);
-  return result;
-}
 
-const Field3D Grad2_par2new(const Field3D& f) {
-  /*
-   * This function implements d2/dy2 where y is the poloidal coordinate theta
-   */
+  Field3D Grad2_par2new(const Field3D& f) {
+    /*
+     * This function implements d2/dy2 where y is the poloidal coordinate theta
+     */
 
-  TRACE("Grad2_par2new( Field3D )");
+    TRACE("Grad2_par2new( Field3D )");
 
-  Field3D result = D2DY2(f);
+    Field3D result = D2DY2(f);
 
 #if BOUT_USE_TRACK
-  result.name = "Grad2_par2new(" + f.name + ")";
+    result.name = "Grad2_par2new(" + f.name + ")";
 #endif
 
-  return result;
-}
-
-const Field2D N0tanh(BoutReal n0_height, BoutReal n0_ave, BoutReal n0_width,
-                     BoutReal n0_center, BoutReal n0_bottom_x) {
-  Field2D result;
-  result.allocate();
-
-  BoutReal Grid_NX, Grid_NXlimit; // the grid number on x, and the
-  BoutReal Jysep;
-  mesh->get(Grid_NX, "nx");
-  mesh->get(Jysep, "jyseps1_1");
-  Grid_NXlimit = n0_bottom_x * Grid_NX;
-  output.write("Jysep1_1 = {:d}   Grid number = {:e}\n", int(Jysep), Grid_NX);
-
-  if (Jysep > 0.) { // for single null geometry
-
-    BoutReal Jxsep, Jysep2;
-    mesh->get(Jxsep, "ixseps1");
-    mesh->get(Jysep2, "jyseps2_2");
-
-    for (int jx = 0; jx < mesh->LocalNx; jx++) {
-      BoutReal mgx = mesh->GlobalX(jx);
-      BoutReal xgrid_num = (Jxsep + 1.) / Grid_NX;
-      // output.write("mgx = {:e} xgrid_num = {:e}\n", mgx);
-      for (int jy = 0; jy < mesh->LocalNy; jy++) {
-        int globaly = mesh->getGlobalYIndex(jy);
-        // output.write("local y = {:d};   global y: {:d}\n", jy, globaly);
-        if (mgx > xgrid_num || (globaly <= int(Jysep) - 2) || (globaly > int(Jysep2) + 2))
+    return result;
+  }
+
+  Field2D N0tanh(BoutReal n0_height, BoutReal n0_ave, BoutReal n0_width,
+                 BoutReal n0_center, BoutReal n0_bottom_x) {
+    Field2D result;
+    result.allocate();
+
+    BoutReal Grid_NX, Grid_NXlimit; // the grid number on x, and the
+    BoutReal Jysep;
+    mesh->get(Grid_NX, "nx");
+    mesh->get(Jysep, "jyseps1_1");
+    Grid_NXlimit = n0_bottom_x * Grid_NX;
+    output.write("Jysep1_1 = {:d}   Grid number = {:e}\n", int(Jysep), Grid_NX);
+
+    if (Jysep > 0.) { // for single null geometry
+
+      BoutReal Jxsep, Jysep2;
+      mesh->get(Jxsep, "ixseps1");
+      mesh->get(Jysep2, "jyseps2_2");
+
+      for (int jx = 0; jx < mesh->LocalNx; jx++) {
+        BoutReal mgx = mesh->GlobalX(jx);
+        BoutReal xgrid_num = (Jxsep + 1.) / Grid_NX;
+        // output.write("mgx = {:e} xgrid_num = {:e}\n", mgx);
+        for (int jy = 0; jy < mesh->LocalNy; jy++) {
+          int globaly = mesh->getGlobalYIndex(jy);
+          // output.write("local y = {:d};   global y: {:d}\n", jy, globaly);
+          if (mgx > xgrid_num || (globaly <= int(Jysep) - 2)
+              || (globaly > int(Jysep2) + 2))
```

Comment:
warning: statement should be inside braces [readability-braces-around-statements]

```cpp
|| (globaly > int(Jysep2) + 2))
                               ^
                                {
```